packages/components/src/List/Toolbar/ItemsNumber/ItemsNumber.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import ItemsNumber from './ItemsNumber.component';
+
+const props = {
+	id: 'list-items-number',
+	totalItems: 42,
+};
+describe('ItemsNumber', () => {
+	it('should render total number of items', () => {
+		// when
+		const wrapper = shallow(<ItemsNumber {...props} />);
+
+		// then
+		expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Please don't use snapshots for that. Snapshots should be used only to check the overall markup. You want to check the text, select it via enzyme and assert the expected value via jest please.